### PR TITLE
Add and correct german translations

### PR DESCRIPTION
**Pull request by ryanthara**
Add missing german translation strings and correct a few not so good translated ones.

### CODE REVIEW

File: Interfaces/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -425,34 +425,34 @@
...
-"Include anonymous system profile when checking for updates text" = "This helps Vienna development by letting us know what versions of Mac OS X are most popular amongst our users.";
-"Error: Feed not found!" = "Error: Feed not found!";
-"Loading" = "Loading";
-"Plugin could not be installed" = "Plugin could not be installed";
-"Vienna failed to install the plugin." = "Vienna failed to install the plugin.";
-"Queue" = "Queue";
-"Forcing Refresh subscriptions..." = "Forcing Refresh subscriptions...";
-"Improper infinitely looping URL redirect to %@" = "Improper infinitely looping URL redirect to %@";
+"Plugin installed" = "Plugin installiert";
+"A new plugin has been installed. It is now available from the menu and you can add it to the toolbar." = "Ein neues Plugin wurde installiert. Es ist nun aus dem Menü verfügbar und kann zur Toolbar hinzugefügt werden.";
```
Comment (Eitot) [inline Interfaces/de.lproj/Localizable.strings]: Should be "im Menü". Toolbar is "Symbolleiste" in German.

File: Interfaces/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -425,34 +425,34 @@
...
+"No thanks" = "Nein Danke";
+"Include anonymous system profile when checking for updates?" = "Anonymisierte Systemprofile beim Suchen nach Aktualisierungen einfügen?";
+"Include anonymous system profile when checking for updates text" = "Dies hilft den Vienna-Entwicklern herauszufinden, welche Mac OS X Version unsere Nutzer verwenden.";
+"Error: Feed not found!" = "Fehler: Feed nicht gefunden!";
+"Loading" = "Laden";
+"Plugin could not be installed" = "Plugin konnten nicht installiert werden";
+"Vienna failed to install the plugin." = "Vienna konnte das Plugin nicht installieren.";
+"Queue" = "Warteschlange";
+"Forcing Refresh subscriptions..." = "Erzwinge Aktualisierung der Abonnements...";
+"Improper infinitely looping URL redirect to %@" = "Unsaubere unendliche URL verweist auf %@";
```
Comment (Eitot) [inline Interfaces/de.lproj/Localizable.strings]: This reads weird. This message means that the URL is stuck in a redirect loop.

File: Interfaces/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -425,34 +425,34 @@
...
+"Error retrieving RSS Icon:" = "Fehler beim Erhalt des RSS-Icon:";
+"RSS Icon not found!" = "RSS-Icon nicht gefunden!";
 
 /*Added in 3.0.5 */
-"Redirection attempt treated as temporary for safety concern" = "Redirection attempt treated as temporary for safety concern";
+"Redirection attempt treated as temporary for safety concern" = "Verbindungsversuch aus Sicherheitsgründen nur temporär möglich";
 
 /*Added in 3.0.7 */
-"No articles in feed" = "No articles in feed";
+"No articles in feed" = "Keine Artikel im feed";
```
Comment (Eitot) [inline Interfaces/de.lproj/Localizable.strings]: Feed should be capitalised.

**Comment by ryanthara** [comment]
Hello. 

I try to implement the changes and use your tips for the next time. Thank you for your advices and the support of my translation try. Appreciated.

**Comment by josh64x2** [comment]
@ryanthara this can probably stay Open while you make your changes.
Thank you for contributing 😄

**Comment by josh64x2** [comment]
@Eitot fyi more questions posted over at https://forums.cocoaforge.com/viewtopic.php?f=18&t=26945&p=139861

**Pull request closed without merging**